lib/tito.rb

```diff
@@ -0,0 +1,29 @@
+require "httparty"
+require_relative "tito/event"
+require_relative "tito/null_event"
+
+class Tito
+  include HTTParty
+
+  base_uri "https://api.tito.io/railsbridge-boston"
+
+  def event_list
```

Comment:
Yeah, I think that makes sense. That’s what I had done with Eventbrite originally. Fixing!